python/ray/train/_internal/state/schema.py

```diff
@@ -5,13 +5,21 @@
 from ray.dashboard.modules.job.pydantic_models import JobDetails
 from ray.util.annotations import DeveloperAPI
 
+MAX_ERROR_STACK_TRACE_LENGTH = 3000
+
+
+@DeveloperAPI
+class TrainRunError(BaseModel):
+    stack_trace: str
+    failed_rank: int
+
 
 @DeveloperAPI
 class RunStatusEnum(str, Enum):
     """Enumeration for the status of a train run."""
 
-    # The train run has started
-    STARTED = "STARTED"
+    # The train run is running
+    RUNNING = "RUNNING"
```

Comment:
this is backwards incompatible, if we do this, we should update train runs to v3

Reply:
I see. Do we need to update the endpoint url to "/api/train/v3/runs"?

Reply:
yeah, probably lets just switch to v3.

Reply:
since this is the backwards-incompatible change, let's add a new enum value instead of replacing the old one. The old one will be deprecated.

Reply:
Make sense. If so, do we still need to switch to v3?

Reply:
nope, it shouldn't be necessary anymore